content/partials/examples/dropdown.slim

```diff
@@ -798,7 +798,7 @@
       .ui.green.empty.circular.label
       |  Tag D
 .ui.hidden.divider
-|  Show me posts trending
+span Show me posts trending 
```

Comment:
Not sure why you add a `span` here. 

And second, if possible, better to remove the extra space in the end of this line.